src/credit.ts

```diff
@@ -301,9 +303,9 @@ export default class Credit extends FactoryBase {
   }
 
   /**
-   * Reflect withdraws complete status to server after transfer tokens
-   * @param {string} appId
-   * @param {DepositTransaction} transaction
+   * Reflects withdraws complete status to server after transfer tokens
```

Comment:
Oh, this description is my fault. Sorry 😢 
Please change description to `Deposit user's credits to Crypto tokens` or your better sentence.

Reply:
Done! I applied your suggestion. Thanks!